src/tasks/mt.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, path, max_seq_len, max_targ_v_size, name, **kw):
         self.max_targ_v_size = max_targ_v_size
         self.target_indexer = {"words": SingleIdTokenIndexer(namespace=self._label_namespace)}
         self.files_by_split = {split: os.path.join(path, "%s.txt" % split) for
-                               split in ["train", "valid", "test"]}
+                               split in ["train", "val", "test"]}
```

Comment:
Why the change?

Reply:
"val" is consistent with the other tasks; not sure why we were using "valid" before :/